### DISCUSSION TITLE
南极谋杀案（十二）- 蟑螂都市 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#448] [General]
# 南极谋杀案（十二）- 蟑螂都市 - Miao Yu | 于淼

https://yufree.cn/cn/2023/12/27/antarictic-murder/

**Comment by earfanfan** [comment]
字少但是信息量好大，我有一个逻辑上的疑惑，蟑螂都市是近二三十年来才重新发展起来的，并且当前是停留在了以建筑物为单位高度分散自治的状态，跨建筑物的交流通过人工智能来完成，可是为什么能够保持在这种大家都得到了相对的自由却又相对平衡的状态，而没有朝着“分久必合、合久必分”的趋势去变化呢？

生活在其中的人类虽然已经做到了自给自足，但是都是怎么忍住对权力的渴望，而没有出现那种比如三国时期的刘备以仁义征服别人聚拢信仰他的人从而白手起家建立国家，也没有像成吉思汗那种天生征服欲强的领导者角色，或者说也没有把自己当神想要拯救别人的圣母圣父，我猜的方向是，可能都被 AI 事先发掘而阻止了，但如果是这样，AI 就是有倾向的了。

**Reply by earfanfan** [reply]
另，虽然你前面写的我现在差不多忘干净了，但我觉得这一篇达到了你最初提到的一个目标，就是三水你只是写一些很散的片段，但是想象的世界在读者脑子里自动串联起来，这一篇我读的时候脑子里确实产生了一种很完整的世界的感觉。

**Reply by yufree** [reply]
额，为啥会有对权力的渴望？在现代世俗社会，法律与行业规章才是权力实体，为了效率优化与应对实际问题，社会发展方向其实没有那么多选择，放着环境问题不管复活节岛就直接被历史抹除了，而领导者更多是拿来背锅与起象征意义的，为所欲为的权力是农业社会平民对统治者的幻想，就算农业社会统治者乱来后果也是灭国，还可能是被小行星灭国，根本与领导人没任何关系。实质上历史里出现过的政体大都是官僚集体用封建礼法统治，属于思想制约并无实质性制约，刘备的仁义成吉思汗的武力或宗教里的神都是后人为权力合理性延续性找的借口，历史也不过是任人打扮的小姑娘罢了。这里有没有AI都一样，我也没说这里自由平衡，事实上纯利益冲突每天都在以各种形式在不同新港市上演。你的困惑在于总是以过去的逻辑看未来，很多你认为的冲突只是价值观冲突而非实际冲突，真实世界里冲突很难只用一两种价值观去归因，更多是多因素天时地利人和碰出来的。

**Reply by earfanfan** [reply]
楼上后半段我是认同的，历史确实是多因素推动的。前半段“与领导人没任何关系”，倒也确实可以这么说，如果是像你这样一口气把历史的时间尺度拉到辣么长的话，但是如果把时间尺度缩短一点的话，古代中国那种集权制度下，权力顶端的皇权和相权代表人物们做的决定也不能说对历史没撒影响，比如秦朝焚书坑儒、统一度量衡、推行官话，时势造英雄和英雄造时势也只是两种历史观而已。好吧，我发现我前面的留言大概依然没有写清楚，俺再捋捋。

对了，你的小说你做主，我只是溜达过来吐槽。你写的南极谋杀案里看上去就是没有政府而只有联盟的，约等于集权制度完全瓦解，法律和行业规章在蟑螂都市也没有，我想的是在这样分权（建筑自治）和混乱（各种利益冲突）的情况下，难道就不会有人想要重新建立新的资源分配秩序么，从分权走向集权，从混乱走向有序。而你小说中那些自由联盟都市，虽然也没有政府，但是处于分权和有序的情况下，没有领导者角色存在才是合理的。

**Reply by earfanfan** [reply]
嗷，我忽然明白原来的困惑了，其实就是在分权和混乱的情况下，怎么没人想着用集权的方式重新建立秩序。我确实是用过去的逻辑看的，历史上就是集结军队直接用暴力建立集权制度的国家，然后重新划分资源分配的秩序。但我还是困惑，为撒在你的未来小说里，暴力的作用好像很小，二三十年的时间里，也没有把小型暴力集结成大型暴力。

**Reply by yufree** [reply]
看开了，第一篇里战争都已经是无人战争了，暴力这种原始解决问题的方案在追求效率的体制下性价比太低，实力差距大打也白打，差距小靠运气，还不如猜拳来的直接，做成南极战争基本已经娱乐化了。另外我没说设定里没有集权，也没说个人作用小，个人与时势在我看来并不对立，个人也是时势一部分而已，对立的只是抽象的概念而不是具体的历史。另外你说的焚书坑儒统一度量衡还有推行官话本就是帝国构建的基础，就算秦始皇不做别人也会做，换到石器时代，就算他想做没有人认同也没法推行下去，就人类世界这个草台班子而言，个体没那么重要。
